Xamarin.Forms.ControlGallery.GTK/Program.cs

```diff
@@ -21,6 +21,7 @@ static void Main(string[] args)
             GtkOpenGL.Init();
             GtkThemes.Init();
             Gtk.Application.Init();
+            Forms.SetFlags("CarouselView_Experimental", "CollectionView_Experimental");
```

Comment:
The experimental label `"CollectionView_Experimental"` is no longer necessary (4.3). Currently, the CollectionView sample would not fail using the master branch.

Reply:
I found this flag in ExperimentalFlags.cs. Should I delete CollectionViewExperimental?

Reply:
No, it is not necessary in this case.